stats/thresholds_parser_test.go

```diff
@@ -0,0 +1,144 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2021 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package stats
+
+import (
+	"reflect"
+	"testing"
+)
+
+func Test_parseThresholdExpression(t *testing.T) {
```

Comment:
Is there a particular reason you decided to go with `Test_xxx` instead of what seems to be the standard `TestXxx` naming that we've been using through k6? 

I am not really particularly for one or the other, but I prefer if there is one that we stick to instead of it being different everywhere ;)

Reply:
Good catch!

We had this discussion internally a while ago, and really decided that test names shouldn't have an underscore separator, indeed.  I think these tests predate that discussion (if not, I'll blame VSCode's snippets instead).